pages/_app.tsx

```diff
@@ -1,8 +1,56 @@
 import 'styles/index.scss'
 import type { AppProps } from 'next/app'
+import Head from 'next/head'
```

Comment:
You import `Head` from `'next/head'` but where is `'next/head'`?

Reply:
anything imported from `next/*` is from the nextJS package